backend/backend/views.py

```diff
@@ -72,6 +75,16 @@ def get(self, request, *args, **kwargs):
 
         return ApiResponse({"tokens": [_bearer_token_dict(token, include_payload=False) for token in tokens]})
 
+    def delete(self, request, *args, **kwargs):
+        try:
+            token = BearerToken.objects.get(token_id=request.GET.get("id"))
+            if request.user == token.user:
+                token.delete()
+                return HttpResponse("Token removed", status=status.HTTP_200_OK)
+        except ObjectDoesNotExist:
```

Comment:
```suggestion
        except BearerToken.ObjectDoesNotExist:
```
and remove import on top.

You may also want to cover `BearerToken.MultipleObjectsReturned` (not expected though)